docs/graphs/graph-sdk-example.md

```diff
@@ -0,0 +1,289 @@
+
+Vertex example in `groceryCustomers` collection:
+
+```json
+{
+    "_key": "C01",
+    "name": "John Doe"
+},
+...
+```
+
+Vertex example in `groceryItems` collection:
+
+```json
+{
+    "_key": "P03",
+    "name": "pineapple"
+},
+...
+```
+
+Graph edge example in `groceryOrders` collection (Connection between customer C01 and item P03):
+
+```json
+{
+    "_from": "groceryCustomers/C01",
+    "_key": "4117657795",
+    "_to": "groceryItems/P03"
+},
+...
+```
+
+## Import the example data
+
+If you want to try out the example, you can import the data into your Macrometa GDN account.
+Data can be found in the `sample-data` folder of this [repository](https://github.com/Macrometacorp/macrometa-graph-example).
+
+- Create two document collections in Macrometa GDN using console.
+    - `groceryItems`
+    - `groceryCustomers`
+
+- Create one edge collection in Macrometa GDN using Macrometa console.
+    - `groceryOrders`
+
+- Import test data from sample-data folder to created collections using console. Name of the files are the same as the collection names.
+
+## How to create the graph on Macrometa GDN
+Now that we have the data imported, we can create the graph on Macrometa GDN using javascript SDK.
+
+### Prerequisites
+
+- A free trial [Macrometa account](https://auth-play.macrometa.io/).
+- A Node.js 14+ installation on your local machine. You can download it from [here](https://nodejs.org/en/download/).
+- Install the Macrometa [javascript SDK](../sdks/install-sdks.md).
+
+:::note
+To check your Node.js version, run the following command in your terminal:
+
+```bash
+node -v
+```
+:::
+
+### Create a graph
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+
+```js
+const jsc8 = require("jsc8");
+
+client = new jsc8({
+  url: "https://play.paas.macrometa.io",
+  apiKey: "xxxxx",
+  fabricName: "_system",
+});
+
+// Please be sure that edge collections and vertices have been created before running this script.
+async function createGraph() {
+  const response = await client.createGraph("grocery-graph", {
+    edgeDefinitions: [
+      {
+        // Edge collection name --> This collection holds relationships between vertices
+        collection: "groceryOrders",
+        // Vertex collection that is used as the start vertex of the edge
+        from: ["groceryCustomers"],
+        // Vertex collection that is used as the start vertex of the edge
+        to: ["groceryItems"],
+      },
+    ],
+  });
+  console.log(response);
+}
+createGraph();
+```
+
+</TabItem>
+
+</Tabs>
+
+After we have created the graph, we can start querying it.
+
+##  How to query the graph
+
+### Query explanation
+Query for the recommendations for a customer can be done using the following query:
```

Comment:
```suggestion

Query for the recommendations for a customer can be done using the following query:
```

Reply:
I don't think this section is a query explanation, more like Query the Graph or Run Graph Traversal Query.